packages/fxa-settings/src/pages/Signin/SigninBounced/index.test.tsx

```diff
@@ -7,9 +7,10 @@ import { screen } from '@testing-library/react';
 import { renderWithRouter } from '../../../models/mocks';
 // import { getFtlBundle, testL10n } from 'fxa-react/lib/test-utils';
 // import { FluentBundle } from '@fluent/bundle';
-import SigninBounced from '.';
+import SigninBounced, { viewName } from '.';
 import { logPageViewEvent } from '../../../lib/metrics';
```

Comment:
Why do we use `usePageViewEvent` in some places and `logPageViewEvent` in others?

Reply:
@millsoper good catch, we should be using `usePageViewEvent`. `logPageViewEvent` is the non-hook version which I remember having to use at least one other time because I couldn't use a hook for some reason, but we should prefer `usePageViewEvent` where we can since it only fires on initial render.

Reply:
ohhhhh gotcha!